src/main/java/games/strategy/engine/framework/startup/mc/ServerModel.java

```diff
@@ -61,13 +61,21 @@
 
 public class ServerModel extends Observable implements IMessengerErrorListener, IConnectionChangeListener {
   public static final RemoteName SERVER_REMOTE_NAME =
-      new RemoteName("games.strategy.engine.framework.ui.ServerStartup.SERVER_REMOTE", IServerStartupRemote.class);
+      new RemoteName("games.strategy.engine.framework.uiComponent.ServerStartup.SERVER_REMOTE", IServerStartupRemote.class);
```

Comment:
Checkstyle: "Line is longer than 120 characters (found 124)."

Reply:
Bump

Reply:
will tackle these, thanks for bump to get back on list.